playbooks/check-for-config-changes.yml

```diff
@@ -0,0 +1,161 @@
+---
+- hosts: mons
+  become: True
+
+  vars:
+    _mon_handler_called: False
+    _osd_handler_called: False
+    _mds_handler_called: False
+    _rgw_handler_called: False
+    _mgr_handler_called: False
+
+  roles:
+    - ceph-defaults
+    - ceph-fetch-keys
+
+- hosts:
+  - mons
+  - agents
+  - osds
+  - mdss
+  - rgws
+  - nfss
+  - restapis
+  - rbdmirrors
+  - clients
+  - mgrs
+  - iscsi-gws
+
+  gather_facts: false
+
+  vars:
+    delegate_facts_host: True
+    _mon_handler_called: False
+    _osd_handler_called: False
+    _mds_handler_called: False
+    _rgw_handler_called: False
+    _mgr_handler_called: False
+
+  pre_tasks:
+
+    - name: get the name of the existing ceph cluster
+      shell: |
+        basename $(grep --exclude '*.bak' -R fsid /etc/ceph/ | egrep -o '^[^.]*' | head -n 1)
+      changed_when: false
+      register: cluster_name
+
+    - name: "stat {{ cluster_name.stdout }}.conf"
+      stat:
+        path: "/etc/ceph/{{ cluster_name.stdout }}.conf"
+      register: ceph_conf_stat
+
+    # Creates a backup of original ceph conf file in 'cluster_name-YYYYMMDDTHHMMSS.conf.bak' format
+    - name: "make a backup of original {{ cluster }}.conf"
+      copy:
+        src: "/etc/ceph/{{ cluster }}.conf"
+        dest: "/etc/ceph/{{ cluster }}-{{ ansible_date_time.iso8601_basic_short }}.conf.bak"
+        remote_src: true
+        owner: "{{ ceph_conf_stat.stat.pw_name }}"
+        group: "{{ ceph_conf_stat.stat.gr_name }}"
+        mode: "{{ ceph_conf_stat.stat.mode }}"
+
+    - name: gather facts
```

Comment:
Makes sense.  I just added that block from take-over-existing-cluster.yml because I thought it was a good idea, but had not tested the placement.  I'll move it.